UN-OCHA/gho-site
# GHO-40: ensure each paragraph gets automatically cleared of floats

# GHO-40

Initial attempt to resolve issue with paragraphs not clearing floats in between each other. If you can quickly figure out why the clearfix isn't working I'm happy to go in that direction, but I couldn't figure it out in my initial investigation and figured this was just as good.

## Testing

- Purely theming. No config or anything to import.
- Add a two-column layout to an Article with one paragraph of text in each column.
- Add another paragraph below the layout and see the effect.
